scripts/EN/Harry

```diff
@@ -3705,9 +3705,9 @@ LoadBackground(0x0); // 0x484e: 00001069
 DrawBackground(); // 0x4854: 00000065
 PlayMusic(0x6e, 0x1); // 0x4858: 0000206c
 PlaySecondSoundFx(0xc); // 0x4860: 00001073
-ShowText((1333)"Inside the storehouse nothing important looking stands out. I carefully edge between the simple metal shelves that have been placed here in a specific arrangement. However, nothing is here that makes this room worthy of being called a storehouse."); // 0x4866: 0000100c
+ShowText((1333)"Inside the storeroom nothing important looking stands out.¥nW2I carefully edge between the simple metal shelves W1that have been placed here in a specific arrangement.¥nW2However, nothing is here that makes this room worthy of being called a storeroom."); // 0x4866: 0000100c
```

Comment:
Changed to Storeroom to match the PS1 map. (Thanks to Hector for noting this on other rooms!)